### DISCUSSION TITLE
need to id increment numbers.

### DISCUSSION THREAD

**Discussion by mani1430** [Power-Components/livewire-powergrid#387] [Ideas]
2. need to id increment numbers.

**Comment by luanfreitasdev** [comment]
duplicated text: https://github.com/Power-Components/livewire-powergrid/issues/366

**Comment by luanfreitasdev** [comment]
Please provide more information as described in the issue outline

**Reply by mani1430** [reply]
1. How I add the $loop->index values in ID column. I don't want to use database table primary id values in the table row number.

2. I need to use encrypted URL , So please help me to how to encrypt the edit route parameter.

please help me.

like => Button::add('edit')
->target('')
->caption('')
->class(' cursor-pointer text-sky-600 px-3 py-2.5 m-1 rounded text-sm')
->route('customer.edit', ['url' => [ 'id' , Crypt::encrypt($id);]]),

how I don't have any idea , please help me.

3. Need to place Actions buttons near to id columns in the table.

**Reply by mani1430** [reply]
Help me please , I don't have any idea.

**Comment by mani1430** [comment]
How do I remove the tailwind dark mode classes.
